packages/validator/src/services/indices.ts

```diff
@@ -101,15 +125,40 @@ export class IndicesService {
     ApiError.assert(res, "Can not fetch state validators from beacon node");
 
     const newIndices = [];
+
+    const allValidatorStatuses = new MapDef<SimpleValidatorStatus, number>(() => 0);
+
     for (const validatorState of res.response.data) {
+      // Group all validators by status
+      const status = statusToSimpleStatusMapping(validatorState.status);
+      allValidatorStatuses.set(status, allValidatorStatuses.getOrDefault(status) + 1);
+
       const pubkeyHex = toHexString(validatorState.validator.pubkey);
       if (!this.pubkey2index.has(pubkeyHex)) {
-        this.logger.debug("Discovered validator", {pubkey: pubkeyHex, index: validatorState.index});
+        //#TODO: We want to include the fee_recipient here, but it's not available in the API
```

Comment:
general thing, I really dislike TODOs in code without an according issue, maybe can introduce something like `TODO(#<issue_number>)` , this approach is for example used by Prysm and I think it is a good idea because from my experience TODOs a lot the time are just excuses not to do things right away and a lot of the time TODOs are there for years...

If we connect an issue to the TODO we can keep track of it and if we close the issue because the problem was resolved or it is no longer relevant then we can also remove the TODO in the code.

Reply:
Good point I will link the issue

Reply:
Still questionable to me if we need TODOs here at all or if the issues are sufficient and if the issues are not sufficient maybe we need to add more context to them instead.

To be fair, I really hate TODOs with a passion, so that might just be me. In my previous projects I would not allow a branch to be merged into the main branch until all TODOs were resolved.

Reply:
The comment is to guide the implementation details which are also mentioned to a degree in the issue, but there needs to be a wider discussion on how to achieve this which will happen on the issue.

Reply:
IMO the TODOs aren't really needed and the issues would be sufficient.
@maschad and I have discussed this PR and the related issues at length and I feel he has an understanding of what needs to be changed without tagging these lines explicitly as TODO.